git_machete/client.py

```diff
@@ -1583,10 +1583,13 @@ def has_any_fork_point_override_config(self, branch: LocalBranchShortName) -> bo
                 self.__git.get_config_attr_or_none(self.config_key_for_override_fork_point_while_descendant_of(branch))) is not None
 
     def __get_fork_point_override_data(self, branch: LocalBranchShortName) -> Optional[ForkPointOverrideData]:
+        to, while_descendant_of = None, None
```

Comment:
Pls provide a test that would've failed before this fix, but passes successfully after the fix (for the sake of the Sacred Coverage)